bundle/python/warning_test.go

```diff
@@ -0,0 +1,195 @@
+	b := &bundle.Bundle{
+		Config: config.Root{
+			Resources: config.Resources{
+				Jobs: map[string]*resources.Job{
+					"job1": {
+						JobSettings: &jobs.JobSettings{
+							JobClusters: []jobs.JobCluster{
+								{
+									JobClusterKey: "cluster1",
+									NewCluster: &compute.ClusterSpec{
+										SparkVersion: "12.2.x-scala2.12",
+									},
+								},
+								{
+									JobClusterKey: "cluster2",
+									NewCluster: &compute.ClusterSpec{
+										SparkVersion: "13.1.x-scala2.12",
+									},
+								},
+							},
+							Tasks: []jobs.Task{
+								{
+									TaskKey:         "key1",
+									PythonWheelTask: &jobs.PythonWheelTask{},
+									JobClusterKey:   "cluster1",
+									Libraries: []compute.Library{
+										{Whl: "./dist/test.whl"},
+									},
+								},
+								{
+									TaskKey:         "key2",
+									PythonWheelTask: &jobs.PythonWheelTask{},
+									JobClusterKey:   "cluster2",
+									Libraries: []compute.Library{
+										{Whl: "./dist/test.whl"},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+	}
+
+	require.True(t, hasIncompatibleWheelTasks(context.Background(), b))
+}
+
+func TestNoIncompatibleWheelTasks(t *testing.T) {
+	b := &bundle.Bundle{
+		Config: config.Root{
+			Resources: config.Resources{
+				Jobs: map[string]*resources.Job{
+					"job1": {
+						JobSettings: &jobs.JobSettings{
+							JobClusters: []jobs.JobCluster{
+								{
+									JobClusterKey: "cluster1",
+									NewCluster: &compute.ClusterSpec{
+										SparkVersion: "12.2.x-scala2.12",
+									},
+								},
+								{
+									JobClusterKey: "cluster2",
+									NewCluster: &compute.ClusterSpec{
+										SparkVersion: "13.1.x-scala2.12",
+									},
+								},
+							},
+							Tasks: []jobs.Task{
+								{
+									TaskKey:         "key1",
+									PythonWheelTask: &jobs.PythonWheelTask{},
+									NewCluster: &compute.ClusterSpec{
+										SparkVersion: "12.2.x-scala2.12",
+									},
+									Libraries: []compute.Library{
+										{Whl: "/Workspace/Users/me@me.com/dist/test.whl"},
+									},
+								},
+								{
+									TaskKey:         "key2",
+									PythonWheelTask: &jobs.PythonWheelTask{},
+									NewCluster: &compute.ClusterSpec{
+										SparkVersion: "13.3.x-scala2.12",
+									},
+									Libraries: []compute.Library{
+										{Whl: "./dist/test.whl"},
+									},
+								},
+								{
+									TaskKey:         "key3",
+									PythonWheelTask: &jobs.PythonWheelTask{},
+									NewCluster: &compute.ClusterSpec{
+										SparkVersion: "12.2.x-scala2.12",
+									},
+									Libraries: []compute.Library{
+										{Whl: "dbfs:/dist/test.whl"},
+									},
+								},
+								{
+									TaskKey:         "key4",
+									PythonWheelTask: &jobs.PythonWheelTask{},
+									JobClusterKey:   "cluster1",
+									Libraries: []compute.Library{
+										{Whl: "/Workspace/Users/me@me.com/dist/test.whl"},
```

Comment:
The test confirms that this should be fine and not trigger a warning.

Is this because it's an abs path and therefore we don't care about it?